cmd/bblfshd/main.go

```diff
@@ -115,10 +117,15 @@ func main() {
 		os.Exit(1)
 	}
 
-	parsedBuild, err := time.Parse("2006-01-02T15:04:05-0700", build)
+	parsedBuild, err := time.Parse(daemon.BuildDateFormat, build)
 	if err != nil {
-		logrus.Errorf("wrong date format for build: %s", err)
-		os.Exit(1)
+		logrus.Errorf("wrong date format for this build: %s", err)
+		if build == defaultBuild {
+			build = time.Now().Format(daemon.BuildDateFormat)
```

Comment:
Why not leave it as `undefined`, but skip an error?

Reply:
because interface of the NewDaemon has changed and it requires parsed `time.Time` no?

Reply:
Sure, but you still pass parsed time (zero `time.Time`) to `NewDaemon` in this case. It doesn't matter what `build` is.

Reply:
Oh, that is a bug, thank you for catching!

Reply:
addressed in parsedBuild